release/air_tests/horovod/workloads/horovod_tune_test.py

```diff
@@ -89,9 +89,10 @@ def train_loop_per_worker(config):
                         epoch=epoch,
                     )
                 )
-            else:
-                checkpoint = None
-            session.report(dict(loss=running_loss / epoch_steps), checkpoint=checkpoint)
+                session.report(
+                    dict(loss=running_loss / epoch_steps), checkpoint=checkpoint
+                )
```

Comment:
wrong place?
move to line 91?

Reply:
Not sure what you mean - I believe it's in the right location? It now only reports when we also save a checkpoint

Reply:
ah sorry, read the diff wrong. 
Yeah, this looks good!